### PR TITLE
Reworked footer and linked text to proper modals

### PR DESCRIPTION
**Pull request by linca-tudor**
### GDPR and TCs modals

Reworked the modals for the TCs and GDPR Policy. They are accessible from the webpage footer.
The register section also has a link to the GDPR Policy included in the checkbox label.

#### The Footer looks now as follows:

Screenshot 2023-02-23 at 16 50 17

#### The GDPR modal:

Screenshot 2023-02-23 at 16 50 52

#### The TCs modal:  

Screenshot 2023-02-23 at 16 52 08

Acessing the GDPR modal from the form checkbox label (Accent color "GDPR" word)

Screenshot 2023-02-23 at 16 52 28

### CODE REVIEW

**Comment by linca-tudor** [comment]
[Create and fill terms and conditions modal](https://trello.com/c/6bh0KUbd/9-create-and-fill-terms-and-conditions-modal)

File: src/config/Strings.js [hunk 1]
```diff
@@ -68,4 +68,15 @@ export const Strings = {
     mainTitle: 'Ne găsești aici',
     backgroundTitle: 'Contact',
   },
+  footer: {
+    copyrightTitle: 'Drepturi de autor © 2023',
+    owner: 'Școala Auto CLD',
```
Comment (anflorea) [inline src/config/Strings.js:73]: Change this to "CLD Auto Drive Consult SRL". Scoala Auto CLD is the comercial name and I think the actual ltd name is more suitable here.

**Review by anflorea — CHANGES_REQUESTED** [review]
Just change that one string, everything else looks good 🚀

File: src/config/Strings.js (hunk 1 above)
Comment (linca-tudor) [reply src/config/Strings.js]: Replaced string as specified
Screenshot 2023-02-24 at 12 51 08

**Review by anflorea — APPROVED** [review]

**Pull request merged**